test/migrators/calc_interpolation/calc_remove_interpolation.hrx

```diff
@@ -2,10 +2,14 @@
 $b: 10;
 $c: 1;
 $d: 5;
-.a { .b: calc($b - #{$c + 1}); }
+//Single interpolation
```

Comment:
Add a space between the `//` and the text here, and a blank line between this and the previous section.